### DISCUSSION TITLE
Users disappear in UI when LDAP enabled

### DISCUSSION THREAD

**Discussion by florian4300** [keycloak/keycloak#24391] [Q&A]
Hello!

I am encoutering a weird issue, that if I enable an LDAP Provider the Users disappear in the UI. 
The users are still in the database, but the Users page is empty. 
![image](https://github.com/keycloak/keycloak/assets/43441416/87e23e0a-4636-4428-8881-b46fa418eaeb)

Furthermore the username from the User Creation Form is missing and I can't create any user, although it is a READ_ONLY User Federation.

![image](https://github.com/keycloak/keycloak/assets/43441416/7290437a-c0c3-4e8e-890c-83fb89c02250)

When I disable the User Federation everything works again.

Is this a Bug in the UI or am I missunderstanding something?

Any help would be appreciated.

**Comment by xBelladonna** [comment]
I can't comment on whether it's a bug or a feature but I've found that if you type `*` or a space into the search box and hit enter, they all show up...
